### ISSUE TITLE
Publicar fotos oficiais

### ISSUE THREAD

**Issue by cecivieira** [pythonNordeste/organizacao-pyne-2023#201]
1. Criar Flirck
2. Divulgar fotos oficiais

**Comment by cecivieira** [comment]
Dado o #197 , falta decidir se o post da foto oficial será fotos cortadas ou outras fotos.
Acho que isso vai depender de tentativa de tratamento de imagem de @tiidadavena 

@tiidadavena quando tu souber a melhor forma de publicar, comenta aqui, por favor

**Comment by cecivieira** [comment]
@rougeth @tiidadavena a conta da PyBR no Flirck é paga? Tem algum desconto de comunidade? 
Queremos fazer uma pra PyNE também

**Comment by rougeth** [comment]
Sim, é paga. Python Brasil atingiu o limite em 2019 se não me engano.

> Tem algum desconto de comunidade?

Parece que tem desconto sim: https://www.flickrhelp.com/hc/en-us/articles/4404058120212-Discounts-for-non-profit-organizations

**Comment by gtcarvalh0** [comment]
Oi pessoal! Alguém consegue me dizer se isso foi feito, se sim, alguém me disponibilidade o link por favor?

**Comment by gtcarvalh0** [comment]
Ah, achei buscando no flickr, obrigado!
